docs/index.rst

```diff
@@ -17,22 +17,22 @@ Key concepts
 
 **Primitives**
 
-Primitives are base level operations that serve as building blocks for many quantum algorithms and applications. The `base primitive interfaces <https://qiskit.org/documentation/apidoc/primitives.html>`__ are defined in Qiskit Terra, and many Qiskit algorithms use the primitives natively. This abstraction allows you to write the same code, using Qiskit algorithms or otherwise, that can run on different quantum hardware or simulators without having to explicitly manage some of the finer details. The primitves offered by `qiskit_ibm_runtime <https://qiskit.org/ecosystem/ibm-runtime/apidocs/ibm-runtime.html>`__ add additional options specific to IBM's service. See `Introduction to primitives <primitives.html>`__ for further details.
+Primitives are base level operations that serve as building blocks for many quantum algorithms and applications. Through these primitives, users are able to obtain high-fidelity results, without the need for detailed hardware knowledge.  This abstraction allows you to write the same code, using Qiskit algorithms or otherwise, that can run on different quantum hardware or simulators without having to explicitly manage aspects such as compilation, optimization, and error suppression / mitigation. The primitives offered by `qiskit_ibm_runtime <https://qiskit.org/ecosystem/ibm-runtime/apidocs/ibm-runtime.html>`__ add additional options specific to IBM's service. See `Introduction to primitives <primitives.html>`__ for further details.
```

Comment:
```suggestion
Primitives are base level operations that serve as building blocks for many quantum algorithms and applications. Through these primitives, users can obtain high-fidelity results, without needing detailed hardware knowledge.  This abstraction allows you to write code, using Qiskit algorithms or otherwise, that can run on different quantum hardware or simulators without having to explicitly manage aspects such as compilation, optimization, and error suppression / mitigation. The primitives offered by `qiskit_ibm_runtime <https://qiskit.org/ecosystem/ibm-runtime/apidocs/ibm-runtime.html>`__ add additional options specific to IBM services. See `Introduction to primitives <primitives.html>`__ for further details.
```